core/extension/gdextension.cpp

```diff
@@ -299,6 +299,8 @@ void GDExtension::_register_extension_class(GDExtensionClassLibraryPtr p_library
 		p_extension_funcs->create_instance_func, // GDExtensionClassCreateInstance create_instance_func; /* this one is mandatory */
 		p_extension_funcs->free_instance_func, // GDExtensionClassFreeInstance free_instance_func; /* this one is mandatory */
 		p_extension_funcs->get_virtual_func, // GDExtensionClassGetVirtual get_virtual_func;
+		nullptr, // GDExtensionClassNotification2 get_virtual_call_data;
+		nullptr, // GDExtensionClassNotification2 call_virtual_func
```

Comment:
These two types in the comment are not correct.

Nitpick, missing semicolon in 2nd line.